src/plugins/dashboard/public/application/index.ts

```diff
@@ -19,4 +19,4 @@
 
 export * from './embeddable';
 export * from './actions';
-export { RenderDeps } from './application';
+export type { RenderDeps } from './application';
```

Comment:
@spalger It seems something broken in our build chain. Shouldn't the optimizer strip it off?

Reply:
No, Babel doesn't have any way to know that the `RenderDeps` export is just a type.